docs/vision-usage.rst

```diff
@@ -0,0 +1,233 @@
+    []
+
+
+Manual Detection
+~~~~~~~~~~~~~~~~
+
+You can call the detection method manually.
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('gs://my-test-bucket/image.jpg')
+    >>> faces = image.detect(type=vision.FACE_DETECTION, limit=10)
+
+Face Detection
+~~~~~~~~~~~~~~
+
+Detecting a face or faces in an image.
+For a list of the possible facial landmarks
+see: https://cloud.google.com/vision/reference/rest/v1/images/annotate#type_1
+
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('./image.jpg')
+    >>> faces = image.detect_faces(limit=10)
+    >>> faces[0].landmarks[0].type
+    'LEFT_EYE'
+    >>> faces[0].landmarks[0].position.x
+    1301.2404
+    >>> faces[0].detection_confidence
+    0.9863683
+    >>> faces[0].joy_likelihood
+    0.54453093
+    >>> faces[0].anger_likelihood
+    0.02545464
+
+
+
+Label Detection
+~~~~~~~~~~~~~~~
+
+Image labels are a way to help categorize the contents of an image.
+If you have an image with a car, person and a dog it, label detection will
+attempt to identify those objects.
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('./image.jpg')
+    >>> labels = image.detect_labels(limit=3)
+    >>> labels[0].description
+    'automobile'
+    >>> labels[0].score
+    0.9863683
+
+
+Landmark Detection
+~~~~~~~~~~~~~~~~~~
+
+The API will attemtp to detect landmarks such as Mount Rushmore and
+the Sydney Opera House. The API will also provide their known geographical
+locations if available.
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('./image.jpg')
+    >>> landmarks = image.detect_landmarks()
+    >>> landmarks[0].description
+    'Sydney Opera House'
+    >>> landmarks[0].locations[0].latitude
+    -33.857123
+    >>> landmarks[0].locations[0].longitude
+    151.213921
+    >>> landmarks[0].bounding_poly.vertices[0].x
+    78
+    >>> landmarks[0].bounding_poly.vertices[0].y
+    162
+
+Logo Detection
+~~~~~~~~~~~~~~
+
+Google Vision can also attempt to detect company and brand logos in images.
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('./image.jpg')
+    >>> logos = image.detect_logos(limit=1)
+    >>> results.logos[0].description
+    'Google'
+    >>> logos[0].score
+    0.9795432
+    >>> logos[0].bounding_poly.vertices[0].x
+    78
+    >>> logos[0].bounding_poly.vertices[0].y
+    62
+
+Safe Search Detection
+~~~~~~~~~~~~~~~~~~~~~
+
+Detecting safe search properties of an image.
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('./image.jpg')
+    >>> safe_search = image.detect_safe_search()
+    >>> safe_search.adult
```

Comment:
By just exposing the choices, we are making people write down and compare against these constants... What people likely want to do is check if it's porn, or violence, or whatever.

```python
image = ...
if image.is_adult():
  print "oops!"
```

can we just make this a "confidence" like we do for faces? ie, the 5 options are 0.02 (very unlikely), 0.25 (unlikely), 0.50 (possible), 0.75 (likely), and 0.99 (very likely) (changed 0 and 100 to 2 and 98 so they are less certain). if we do that, code looks like...

```python
image = ...
if image.get_adult_likelihood() > 0.5:
  print 'oops!'
```